src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -1001,5 +1003,13 @@ public static void registerShaders(RegisterShadersEvent event) throws IOExceptio
                 rendertypeEntityTranslucentUnlitShader = p_172645_;
             });
         }
+
```

Comment:
Move to ForgeSpawnEggItem, private method, so that getModEggs() does not have to be public.